lms/static/js/accessible_components/StatusBarAlert.jsx

```diff
@@ -4,7 +4,7 @@ Wrapper for React/Paragon accessible status bar
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import StatusAlert from '@edx/paragon/src/StatusAlert/index.jsx';
+import { StatusAlert } from '@edx/paragon/static';
```

Comment:
@arizzitano in order to get rid of the ExtractTextPlugin and *.css/*.scss rules in the prod webpack config, I had to change this page over to use the static build. I tested it out locally and it looks like it still works fine.

Reply:
Ahh, that's great news!! Thanks for picking up this task.